modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,451 @@
++
+[source,yaml]
+----
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+    labels:
+        app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+    name: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+    annotations:
+specialresource.openshift.io/wait: "true"
+spec:
+    nodeSelector:
+        node-role.kubernetes.io/worker: ""
+    runPolicy: "Serial"
+    triggers:
+        - type: "ConfigChange"
+        - type: "ImageChange"
+    source:
+        git:
+            ref: {{.Values.git.ref}}
+            uri: {{.Values.git.uri}}
+        type: Git
+    strategy:
+        dockerStrategy:
+            dockerfilePath: Dockerfile.SRO
+            buildArgs:
+                - name: "IMAGE"
+                value: {{ .Values.driverToolkitImage  }}
+                {{- range $arg := .Values.buildArgs }}
+                - name: {{ $arg.name }}
+                value: {{ $arg.value }}
+                {{- end }}
+                - name: KVER
+                value: {{ .Values.kernelFullVersion }}
+    output:
+        to:
+            kind: DockerImage
+            name: {{.Values.registry}}/{{.Values.specialResourceModule.metadata.name}}-{{.Values.groupName.driverContainer}}:{{.Values.kernelFullVersion}}
+----
+.. Save this YAML template for the ACM policy in the `templates` directory as `0002-policy.yaml`.
+... For a connected environment:
++
+[source,yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    name: policy-{{.Values.specialResourceModule.metadata.name}}-ds
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST-CSF
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: config-{{.Values.specialResourceModule.metadata.name}}-ds
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                        objectDefinition:
+                        apiVersion: v1
+                        kind: Namespace
+                        metadata:
+                            name: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: mustonlyhave
+                        objectDefinition:
```

Comment:
`objectDefinition` should be at the same level as `complianceType`

Reply:
corrected